modules/summarizing-cluster-specifications-through-clusterversion.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * support/summarizing-cluster-specifications.adoc
+
+[id="summarizing-cluster-specifications-through-clusterversion_{context}"]
+= Summarizing cluster specifications through 'clusterversion'
```

Comment:
Same as other comments, use backticks in headings, not single quotes

Reply:
I have now resolved this throughout the PR.